src/createServer.js

```diff
@@ -1,8 +1,57 @@
-'use strict';
+const http = require('http');
+const url = require('url');
+const fs = require('fs');
 
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}`);
+
+    const pathname = normalizedUrl.pathname.slice(1) || 'index.html';
+
+    res.setHeader('Content-Type', 'text/plain');
+
+    if (!req.url.startsWith('/file')) {
+      res.statusCode = 400;
+
+      res.end();
+
+      return;
+    }
```

Comment:
The condition `!req.url.startsWith('/file')` is redundant because the next condition `!req.url.startsWith('/file/')` will handle cases where the URL does not start with '/file/'. Consider removing this condition to simplify the code.